newIDE/app/src/VariablesList/VariableRow.js

```diff
@@ -8,8 +8,10 @@ import AddCircle from 'material-ui/svg-icons/content/add-circle';
 import SubdirectoryArrowRight from 'material-ui/svg-icons/navigation/subdirectory-arrow-right';
 import TextField from 'material-ui/TextField';
 import IconButton from 'material-ui/IconButton';
+import Reset from 'material-ui/svg-icons/av/replay';
 import muiThemeable from 'material-ui/styles/muiThemeable';
 import styles from './styles';
+import { type variableOrigin } from './VariablesList.flow';
```

Comment:
A type should be PascalCased: `VariableOrigin`. Otherwise, it can confused with a variable, and would prevent us to name a variable `variableOrigin`.

Reply:
that is a very good design. Sorry I missed it